caffe2/operators/tt_linear_op.h

```diff
@@ -55,6 +54,7 @@ class TTLinearOp final : public Operator<Context> {
     auto Y_buf = BlobGetMutableTensor(Y_temp_.get(), Context::GetDeviceType());
     Y_buf->ResizeLike(X);
     Y_buf->CopyFrom(X);
+    Tensor* Y;
```

Comment:
Maybe add an assertion that `d>=1`, because otherwise you're going to get a nasty nullptr exception.